test/e2e/e2e.go

```diff
@@ -340,7 +340,7 @@ func logClusterImageSources() {
 // TODO: These should really just use the GCE API client library or at least use
 // better formatted output from the --format flag.
 
-// Returns master & node image string, or error
+// Returns control plane & node image string, or error
```

Comment:
As stated earlier, maybe update this to be more specific:
`// Returns control plane node & data plane node image string, or error`